src/main/java/at/hannibal2/skyhanni/config/features/fishing/trophyfishing/GeyserOptionsConfig.java

```diff
@@ -0,0 +1,33 @@
+package at.hannibal2.skyhanni.config.features.fishing.trophyfishing;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.HasLegacyId;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorColour;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class GeyserOptionsConfig {
+
+    @Expose
+    @ConfigOption(
+        name = "Hide Geyser Particles When Fishing",
+        desc = "Stops the white geyser smoke particles from rendering if your bobber is near the geyser.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean hideGeyserParticles = true;
```

Comment:
We are already in the Geyser config class, no need to contain the word Geyser in the config names here as well